es/lessons/basics/strings.md

```diff
@@ -0,0 +1,158 @@
+---
+version: 1.1.1
+title: Cadenas
+---
+
+Cadenas, listas de caracteres, Grafemas y puntos de código.
+
+{% include toc.html %}
+
+## Cadenas
+
+Las cadenas en Elixir no son mas que una secuencia de bytes. Vamos a ver un ejemplo:
```

Comment:
I think that _bytes_ should be in _italic_ here.

Reply:
Done